api/openapi-spec/swagger.json

```diff
@@ -7287,6 +7369,88 @@
      }
     ]
    },
+   "/apis/subresources.kubevirt.io/v1alpha3/namespaces/{namespace:[a-z0-9][a-z0-9\\-]*}/virtualmachineinstances/{name:[a-z0-9][a-z0-9\\-]*}/ssh/{interface:[a-z0-9][a-z0-9\\-]*}/{port:[0-9]+}": {
+    "get": {
+     "description": "Open a websocket connection to connect to SSH on the specified VirtualMachineInstance network interface and port.",
+     "operationId": "v1alpha3SSH/withInterfaceName",
+     "responses": {
+      "401": {
+       "description": "Unauthorized"
+      }
+     }
+    },
+    "parameters": [
+     {
+      "uniqueItems": true,
+      "type": "string",
+      "description": "The network interface on the VirtualMachineInstance the SSH connection should be forwarded to.",
+      "name": "interface",
```

Comment:
I would prefer if we don't add this initially in this PR. It may be fine, but in general we don't have the guarantee that `virt-api` can talk to multus or sriov networks. I think it deserves an extra PR.